ci_framework/roles/edpm_prepare/tasks/main.yml

```diff
@@ -220,6 +220,7 @@
       ansible.builtin.uri:
         url: "{{ _cifmw_edpm_prepare_keystone_endpoint_out.stdout | trim }}"
         status_code: "{{ _keystone_response_codes }}"  # noqa: args[module]
+        validate_certs: false
```

Comment:
We probably don't want to hardcode that "false". Care to add a new parameter to that role? like `cifmw_edpm_prepare_verify_tls` or the like?